paper/Astic_Oldenburg_PGI_article_revised_v2.tex

```diff
@@ -0,0 +1,923 @@
+
+In the MT experiment the electric and magnetic fields recorded at the earth’s surface are combined to generate impedances. In this example, data are collected at 25 logarithmically spaced frequencies between $10^{-3}$ and $10^3$ Hz. This generates 50 data composed of the real and imaginary components of the impedance for each frequency. Unbiased Gaussian noise with a standard-deviation of $2\%$ of the data value is added. We represent those observed data in the form of apparent conductivity and phase in Fig. \ref{fig:MT1D_Setup}(c).
+
+
+\subsubsection{Initialization}
+
+We first describe our initialization for both the Tikhonov and PGI algorithms. For the initial geophysical model $\mathbf{m}_0$, we choose a uniform half-space with the same conductivity as the background unit. For the PGI approach the starting membership $\mathbf{z}_0$ is set to the background unit everywhere. Both the Tikhonov and PGI approaches start with the same objective function; the models after the first iteration are thus identical. This helps for comparing results between the two approaches.
+
+For the PGI approach we also need to provide a GMM to describe our prior petrophysical and geological information $\Theta_{\text{prior}}$. We use the exact information from the true earth model for the means and global proportions. For the variances, we use the true variance for the smooth unit. This unit is characterized by a wide distribution and thus it has a large variance. The background and the resistive unit are characterized by single values so their true variances are zero (Dirac distribution). We assign a standard deviation of $0.1~\ln(\text{S/m})$ (or variance of $0.01~[\ln(\text{S/m})]^2$) as an acceptable level of petrophysical noise for those two units. This prior petrophysical distribution can be visualized in Fig. \ref{fig:MT1D_Setup}(b). All the parameters were estimated by taking into account the "volume" of the cells. This is necessary to make the GMM parameters independent of the discretization. The histogram in Fig. \ref{fig:MT1D_Setup}(b) and all subsequent take into account the cell volumes and are thus referred to as weighted histograms.
+
+We set all confidences in the prior petrophysical distribution $\{\mathbf{\zeta},\mathbf{\kappa},\mathbf{\nu}\}$ to unity. Unity confidence parameters mean we have equal trust in the geophysical model and in the prior information when computing the new GMM parameters at each iteration of the petrophysical characterization. Each new estimate of the parameters, at each iteration of the MAP-EM algorithm (equations \ref{pi_update}, \ref{mu_update} and \ref{sig_update}), is thus a simple unweighted average of the observed and priors. This allows us to see how well each parameter of the GMM is recovered without forcing or fixing its value in the inversion.
+
+
+\subsubsection{Comparison of the Tikhonov and PGI results}
+
+\begin{figure*}
+ \centering
+ \includegraphics[width=\textwidth]{MT1D_Results.png}
```

Comment:
A number of figures here are not resolving; they need to be replaced with the full path:
```suggestion
 \includegraphics[width=\textwidth]{Fig/LowRes/MT1D_Results.png}
```